staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -357,7 +357,7 @@ func waitForDelete(params waitForDeleteParams) ([]corev1.Pod, error) {
 			default:
 				return false, nil
 			}
-			return false, nil
```

Comment:
why is it unnecessary?

Reply:
> why is it unnecessary?

upper `select` statement already return anyway, so this return maybe  unreachable.